app/mailers/edit_request_mailer.rb

```diff
@@ -0,0 +1,58 @@
+class EditRequestMailer < ActionMailer::Base
+  default from: 'Annict <no-reply@annict.com>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.